app/models/base_organisation.rb

```diff
@@ -44,4 +44,13 @@ def gmaps4rails_marker_attrs
   def has_been_updated_recently?
     updated_at >= 1.year.ago
   end
+  
+  def slug_candidates
+    # It checks if self.name is nil, because otherwise it breaks -
```

Comment:
hmm, wouldn't it be preferable to have the SetupSlug method be able to handle the nil?  It could just give back a random id in that case, and then we'd never have an error.

I think that might be more robust, and make the code cleaner ...